project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/SettingValueShopInfoDataFixture.php

```diff
@@ -40,14 +40,9 @@ public function load(ObjectManager $manager)
         foreach ($this->domain->getAll() as $domainConfig) {
             $domainId = $domainConfig->getId();
             $locale = $domainConfig->getLocale();
-            $settingValues = [
-                ShopInfoSettingFacade::SHOP_INFO_PHONE_NUMBER => t('+1-234-567-8989', [], 'dataFixtures', $locale),
-                ShopInfoSettingFacade::SHOP_INFO_PHONE_HOURS => t('(Mon - Sat: 9 - 10 a.m. to 8 - 10 p.m.)', [], 'dataFixtures', $locale),
-                ShopInfoSettingFacade::SHOP_INFO_EMAIL => t('no-reply@shopsys.com', [], 'dataFixtures', $locale),
-            ];
-            foreach ($settingValues as $key => $value) {
-                $this->setting->setForDomain($key, $value, $domainId);
-            }
+            $this->setting->setForDomain(ShopInfoSettingFacade::SHOP_INFO_PHONE_NUMBER, t('+1-234-567-8989', [], 'dataFixtures', $locale), $domainId);
+            $this->setting->setForDomain(ShopInfoSettingFacade::SHOP_INFO_PHONE_HOURS, t('(Mon - Sat: 9 - 10 a.m. to 8 - 10 p.m.)', [], 'dataFixtures', $locale), $domainId);
+            $this->setting->setForDomain(ShopInfoSettingFacade::SHOP_INFO_EMAIL, t('no-reply@shopsys.com', [], 'dataFixtures', $locale), $domainId);
```

Comment:
Why have you rewriten this?

I would leave it as it was before and just extract the foreach into another method. What do you think?

Reply:
Because @DavidOstrozlik did not like it before...

Reply:
I'm voting up for something similar to this one bellow

```
updateSettings([
    ShopInfoSettingFacade::SHOP_INFO_PHONE_NUMBER => t('+1-234-567-8989', [], 'dataFixtures', $locale),
    ShopInfoSettingFacade::SHOP_INFO_PHONE_HOURS => t('(Mon - Sat: 9 - 10 a.m. to 8 - 10 p.m.)', [], 'dataFixtures', $locale),
    ShopInfoSettingFacade::SHOP_INFO_EMAIL => t('no-reply@shopsys.com', [], 'dataFixtures', $locale),
])

function updateSettings($settingValues): void
{
    foreach ($settingValues as $key => $value) {
        $this->setting->setForDomain($key, $value, $domainId);
    }
}
```

Reply:
currently, it is shorter and more readable :wink:

Reply:
@s3tezsky I'm not.. its too complicated and long for what it does and has no real added value..